tests/EvmLookupServer/Test_EvmLookupServer.cpp

```diff
@@ -29,21 +29,37 @@ class AbstractServerConnectorMock : public jsonrpc::AbstractServerConnector {
   auto StopListening() -> bool final { return true; }
 };
 
-BOOST_AUTO_TEST_SUITE(lookup_server)
+/**
+ * @brief Default Mock implementation for the evm client
+ */
+class EvmClientMock : public EvmClient {
+ public:
+  EvmClientMock() = default;
+
+  auto OpenServer(bool /*force = false*/) -> bool { return true; };
```

Comment:
The fact that newer C++ allows arrow function declaration/definition, doesn't make it automatically preferred syntax, does it?

Especially given that the current codebase uses the more traditional way, which is *not* declared obsolete in any way, AFAIK.  

Don't want to be judgemental about style, really would want to leave it to the team, but I strongly prefer consistency whatever the decisions is.

Reply:
I understand your point, but this is a c++11 style and that is the current standard that we use. I am afraid that as we don't adopt new language features and styles we will have exceptions on exceptions and will stay in c++99. It also shows that our coding standard is not up-to-date.

Reply:
Which style guide recommends always using newer forms?  The newer forms are available to use when it makes sense, but whether to encourage its use is probably left to the team.

E.g. I've been following this style, and find it useful: https://google.github.io/styleguide/cppguide.html#trailing_return

Same about `auto`, which is great when the type is obscure, and hurts readability otherwise. Just because something is available, doesn't necessarily mean it is better:


> The fundamental rule is: use type deduction only to make the code clearer or safer, and do not use it merely to avoid the inconvenience of writing an explicit type. When judging whether the code is clearer, keep in mind that your readers are not necessarily on your team, or familiar with your project, so types that you and your reviewer experience as unnecessary clutter will very often provide useful information to others. For example, you can assume that the return type of make_unique<Foo>() is obvious, but the return type of MyWidgetFactory() probably isn't.